docs/user/lambda/install.rst

```diff
@@ -0,0 +1,267 @@
+
+As described in section :ref:`geomesa_and_geoserver`, GeoMesa implements a `GeoTools`_-compatible data store.
+This makes it possible to use GeoMesa as a data store in `GeoServer`_. GeoServer's web site includes
+`installation instructions for GeoServer`_.
+
+.. _installation instructions for GeoServer: http://docs.geoserver.org/stable/en/user/installation/index.html
+
+After GeoServer is installed, you may install the WPS plugin if you plan to use GeoMesa processes. The GeoServer
+WPS Plugin must match the version of the GeoServer instance. The GeoServer website includes instructions for
+downloading and installing `the WPS plugin`_.
+
+.. _the WPS plugin: http://docs.geoserver.org/stable/en/user/services/wps/install.html
+
+.. note::
+
+    If using Tomcat as a web server, it will most likely be necessary to
+    pass some custom options::
+
+        export CATALINA_OPTS="-Xmx8g -XX:MaxPermSize=512M -Duser.timezone=UTC \
+        -server -Djava.awt.headless=true"
+
+    The value of ``-Xmx`` should be as large as your system will permit; this
+    is especially important for the Kafka plugin. You
+    should also consider passing ``-DGEOWEBCACHE_CACHE_DIR=/tmp/$USER-gwc``
+    and ``-DEPSG-HSQL.directory=/tmp/$USER-hsql``
+    as well. Be sure to restart Tomcat for changes to take place.
+
+
+To install the GeoMesa Lambda data store as a GeoServer plugin, unpack the contents of the
+``geomesa-lambda-gs-plugin_2.11-$VERSION-install.tar.gz`` file in ``geomesa-lambda_2.11-$VERSION/dist/geoserver/``
+in the binary distribution or ``geomesa-$VERSION/geomesa-lambda/geomesa-lambda-gs-plugin/target/`` in the source
+distribution into your GeoServer's ``lib`` directory (``$VERSION`` = |release|):
+
+If you are using Tomcat:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-lambda_2.11-$VERSION/dist/geoserver/geomesa-lambda-gs-plugin_2.11-$VERSION-install.tar.gz \
+      -C /path/to/tomcat/webapps/geoserver/WEB-INF/lib/
+
+If you are using GeoServer's built in Jetty web server:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-lambda_2.11-$VERSION/dist/geoserver/geomesa-lambda-gs-plugin_2.11-$VERSION-install.tar.gz \
+      -C /path/to/geoserver/webapps/geoserver/WEB-INF/lib/
+
+There are additional JARs for Accumulo, Zookeeper, Hadoop, Thrift and Kafka that you will need to copy to GeoServer's
+``WEB-INF/lib`` directory. The versions required will be specific to your installation. For example, GeoMesa only
+requires Hadoop |hadoop_version|, but if you are using Hadoop 2.5.0 you should use the JARs
+that match the version of Hadoop you are running.
+
+There are scripts in the ``geomesa-lambda_2.11-$VERSION/bin`` directory
+(``install-hadoop-accumulo.sh``, ``install-kafka.sh``) which will install these dependencies to a target directory
+using ``wget`` (requires an internet connection).
+
+.. note::
+
+    You may have to edit ``install-hadoop-accumulo.sh`` and/or ``install-kafka.sh`` to set the
```

Comment:
probably it won't work with any version other than 0.9.x... but the kafka jars aren't bundled anywhere so there's no need to recompile either way